README.rst

```diff
@@ -66,52 +66,107 @@ If you would like to use the Pygame API, from your anaconda environment run::
 
     pip install pygame
 
-Vrep
+CoppeliaSim
 ----
-We support Vrep <=4.0. You will need to download
+We support CoppeliaSim <=4.2. You will need to download
 `Vrep <http://coppeliarobotics.com/previousVersions/>`_ and follow the installation
 instructions.
 
+
+ABR Jaco2
+----
+In addition to simulation of the Kinova Jaco2 in CoppeliaSim and Mujoco, we have an
+API for controlling the real arm. For installation instructions, see the
+`ABR_Jaco2 <https://github.com/abr/abr_jaco2/>`_ repository.
+
 PyDMPs
```

Comment:
the DMP trajectory planner is gone, right? we don't need to mention alternative ways here, that's fine. we should also remove pydmps from any mention in the setup files

Reply:
(i'll do this)